frontend/taipy-gui/src/components/Taipy/FileSelector.spec.tsx

```diff
@@ -34,7 +34,7 @@ describe("FileSelector Component", () => {
     it("displays the right info for string", async () => {
         const { getByText } = render(<FileSelector label="toto" defaultLabel="titi" className="taipy-file-selector" />);
         const elt = getByText("toto");
-        expect(elt.parentElement).toHaveClass("taipy-file-selector");
+        expect(elt.parentElement?.parentElement).toHaveClass("taipy-file-selector");
```

Comment:
Could `elt.parentElement?.parentElement` be null?
That seems to be the case considering the changes elsewhere...

Reply:
ParentElement can always be null but not here :-)